python/requirements-test.txt

```diff
@@ -1,5 +1,4 @@
 cffi
-cython
```

Comment:
Why do you remove this? There's a Cython test.

Reply:
Becuase we usually pass bot requirements files and defining a package twice makes pip unhappy.